src/Tribe/Admin/Views.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__Admin__Views
```

Comment:
doc compilers that are not written in PHP will use that, the one Symfony uses is Java and is not PHP aware.